app/static/src/services/serverConnector.js

```diff
@@ -0,0 +1,26 @@
+export default class ServerConnector {
+    constructor() {
```

Comment:
U can pass path as an argument to contructior
constructor(path = '') {
this.path = path;
}